test/parallel/test-spawn-cmd-named-pipe.js

```diff
@@ -42,10 +42,8 @@ if (!process.argv[2]) {
     assert.fail('Failed to get COMSPEC');
   }
 
-  const args = ['/c', process.execPath, __filename, 'child',
-                '<', stdinPipeName, '>', stdoutPipeName];
-
-  const child = spawn(comspec, args);
+  const child = exec(`"${process.execPath}" "${__filename}" child < ${
```

Comment:
`windowsVerbatimArguments` is not documented. See https://github.com/nodejs/node/blob/cfe7b34058716d9c9dfb143ecc865a30f433db5c/lib/child_process.js#L424-L428.

When you run `exec()` or `spawn()` with `shell: true`, this code path is hit. It's probably what you're looking for.